contentgrid-spring-querydsl/src/main/java/com/contentgrid/spring/querydsl/hibernate/PostgresNormalizeFunctionContributor.java

```diff
@@ -9,5 +9,6 @@ public class PostgresNormalizeFunctionContributor implements FunctionContributor
     public void contributeFunctions(FunctionContributions functionContributions) {
         var returnType = functionContributions.getTypeConfiguration().getBasicTypeForJavaType(String.class);
         functionContributions.getFunctionRegistry().registerPattern("normalize", "normalize(?1, NFKC)", returnType);
+        functionContributions.getFunctionRegistry().registerPattern("contentgrid_normalize", "extensions.contentgrid_prefix_search_normalize(?1)", returnType);
```

Comment:
```suggestion
        functionContributions.getFunctionRegistry().registerPattern("contentgrid_prefix_search_normalize", "extensions.contentgrid_prefix_search_normalize(?1)", returnType);
```

I would not simplify this function down to just contentgrid_normalize, because such a name is a bit too generic.


Additional question: does explicitly prefixing it with `extensions` make it unnecessary to actually have `extensions` in the search path?